pimcore/models/Tool/Email/Blacklist/Listing/Dao.php

```diff
@@ -45,8 +45,13 @@ public function load()
         return $addresses;
     }
 
+    /**
+     * @return int
+     */
     public function getTotalCount()
     {
+        $amount = 0;
```

Comment:
Same as before - please do not alter behaviour in this huge commit, rather add a TODO.